src/test/java/uk/gov/justice/hmpps/prison/util/builders/OffenderBookingBuilder.kt

```diff
@@ -64,7 +71,7 @@ class OffenderBookingBuilder(
     val request =
       RequestForNewBooking.builder().bookingInTime(bookingInTime).cellLocation(cellLocation)
         .fromLocationId(fromLocationId).imprisonmentStatus(imprisonmentStatus).movementReasonCode(movementReasonCode)
-        .prisonId(prisonId).youthOffender(youthOffender).build()
+        .prisonId(prisonId).build()
```

Comment:
move the setting of youth offender to the OffenderProfileDetailsBuilder precisely because it doesn't work correctly via the service